src/ServiceControl.Monitoring.Tests/Infrastructure/IntervalStoreTests.cs

```diff
@@ -162,9 +162,32 @@ public void Intervals_are_returned_in_descending_order()
             Assert.IsTrue(intervalStarts[1] > intervalStarts[2]);
         }
 
-        IntervalsStore AnyStore()
+        IntervalsStore<BreakdownId> AnyStore()
         {
-            return new IntervalsStore(TimeSpan.FromSeconds(5), 127);
+            return new IntervalsStore<BreakdownId>(TimeSpan.FromSeconds(5), 127);
+        }
+
+        class BreakdownId
```

Comment:
There's no generic requirement for the type of breakdown. Why not to use some primitive, like `int`?